controllers/webhook/webhook.go

```diff
@@ -36,25 +37,29 @@ var log = ctrl.Log.WithName("odh-controller-webhook")
 //nolint:lll
 
 type OpenDataHubWebhook struct {
-	client  client.Client
-	decoder *admission.Decoder
+	Client  client.Client
+	Decoder *admission.Decoder
 }
 
-func (w *OpenDataHubWebhook) SetupWithManager(mgr ctrl.Manager) {
+func (w *OpenDataHubWebhook) SetupWithManager(mgr ctrl.Manager) error {
+	// Initialize
+	if mgr == nil {
+		return errors.New("operator manager is nil, should init first")
+	}
+	if w.Client == nil { // replace the old InjectClient() method
+		w.Client = mgr.GetClient()
+	}
+	if w.Decoder == nil { // replace the old InjectDecoder() method
+		w.Decoder = admission.NewDecoder(mgr.GetScheme())
+	}
 	hookServer := mgr.GetWebhookServer()
+	if hookServer == nil {
```

Comment:
Is it possible? IIRC it creates default one if it does not exist. Or it's the change in the new version?